tokens/governance/treasury-spend.md

```diff
@@ -1,97 +1,103 @@
+The happy path of a treasury spend request is as follows:
 
-![Proposal Roadmap](/images/tokens/governance/treasury-proposals/treasury-proposal-roadmap.webp)
+1. **Proposal Submission** - The user submits a proposal to the [Moonbeam Forum](https://forum.moonbeam.network/c/governance/treasury-proposals/8){target=\_blank}
 
-## Submitting your Idea to the Forum {: #submitting-your-idea-to-the-forum }
+2. **Forum Discussion** - The proposal is discussed by the community on the Forum. The ultimate yay/nay decision is determined by the treasury council
 
-It's highly recommended that you preface any proposal with a post on [Moonbeam's Community Forum](https://forum.moonbeam.network){target=\_blank}. You should allow a period of five days for the community to provide feedback on the Moonbeam Forum post. This is especially important for Treasury proposals because of the irrevocable bond required to submit a Treasury spend proposal. You can submit your idea for a Treasury spend proposal by following [the instructions in the Using the Moonbeam Community Forum to Submit a Treasury Proposal guide](https://moonbeam.network/news/using-the-moonbeam-community-forum-to-submit-a-treasury-proposal){target=\_blank}.
+3. **Treasury Approval & Action** - If the Treasury Council agrees, it authorizes the treasury spending and moves the process forward
 
 ![Moonbeam Forum Home](/images/tokens/governance/treasury-proposals/treasury-proposal-1.webp)
 
-## Proposing an Action {: #proposing-an-action }
+## Submitting your Idea to the Forum {: #submitting-your-idea-to-the-forum }
+
+As mentioned in the happy path above, the first step of a treasury proposal is to submit the proposal to the [Moonbeam Forum](https://forum.moonbeam.network/c/governance/treasury-proposals/8){target=\_blank}. To properly submit a treasury spend request, take the following steps:
+
+1. From the [governance section](https://forum.moonbeam.network/c/governance/treasury-proposals/8){target=\_blank}, click **New Topic**. By starting the topic in the governance section, the proposal will come pre-filled with a template to ensure you cover all the necessary points 
+2. Provide a title for the proposal
+3. Enter the contents of the proposal, making sure to cover each of the following categories:
 
-This guide focuses on the mechanics of submitting an on-chain Treasury proposal after you have worked with the Treasury Council to refine your idea. If you haven't yet completed the prior steps of the Treasury proposal process, please take a moment to review the [guidelines of the community Treasury program](https://github.com/moonbeam-foundation/treasury/blob/main/interim/interim_treasury_proposal.md){target=\_blank} and evaluate your standing. Engaging the community and revising your proposal based on feedback received is the most critical piece of your proposal. Submitting the on-chain Treasury proposal is the easy part, demonstrated in the following guide.
+    - **Title** and **Proposal Status** 
```

Comment:
I'm not sure what exactly is trying to be achieved here. I see it two ways, this is either a list item where we follow: `**Title** - description` format. Or this is pointing out UI elements, if it's pointing out UI elements, the text should be exactly as shown in the UI. So **Proposal Status** is not shown per the image. It's close but not exactly. We should be making exact references

Reply:
To follow suit with the rest of the guide, I would probably change this to:

`- **Title** - include the proposal number, its current status, and the title.`